packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -210,6 +212,29 @@ export default class VList {
         this.rootList = null;
     }
 
+    /**
+     * Adds a dummy Li Element that is going to be used to split the VList in the WriteBack function
+     * @param separator The HTML element that indicates when to split the VList
+     */
+    split(separator: HTMLElement) {
+        if (!this.rootList) {
+            throw new Error('rootList must not be null');
+        }
+
+        //Traverse the items of the VList, when the separator is found, add a dummy element
+        for (let index = 0; index < this.items.length; index++) {
+            if (this.items[index].getNode() == separator) {
+                let dummyElement = this.items[index].getNode().cloneNode(false);
+
+                separator.parentElement.append(dummyElement);
```

Comment:
Let's use parentNode instead of parentElement since it doesn't work well in IE

Reply:
And use appendChild instead of append due to IE.

Reply:
Actually, no need to appendChild, just do insertBefore() directly is ok

Reply:
After debug with your code, I think you do not need to clone and append the list node. Instead, you can add some property in VListItem to specify the list number to reset, by default its value is undefined, which means no need to reset. But when its number is different than the start number we get from this list (line 180), we break current list and start a new one.

so that in this function, we just need to find the existing VListItem object from the element you passed in, and set its number (if it is different than the current one). Then let writeBack() handle everything else.

Reply:
I added the new property to the VListItem, I called it "newListStart", with its getter and setter.

What I am having a little issue with is changing the WriteBack function:
![image](https://user-images.githubusercontent.com/8291124/130149311-6e2d7dd3-fc0d-446c-93dd-679d571014c1.png)

In the first change I remove the items when the item.getNewListStart() returns something (I am assiging this value on the "split method"), then just change the parent List's start property. I am not sure if there is any feedback on this change of the WriteBack Function